.openshift-ci/sensor_versions.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""
+Returns the latest patches of the last 4 major versions
+"""
+
+import sys
+import re
+import subprocess
+
+def isReleaseTagGitCLI(version):
```

Comment:
I think mentions of `GitCLI` or just `cli` in function names are just noise. I suggest to remove them unless they are necessary and bring some meaning.

Reply:
Okay I'll rename it. It's basically a legacy name now since I previously had 3 different methods of getting the tags but now there is only 1 so the distinction is unnecessary.